docs/source/howto/codes.rst

```diff
@@ -45,8 +45,92 @@ Design guidelines
 Running external codes
 ======================
 
-`#3987`_
+The process of running an external code through AiiDA involves loading and instantiating the appropriate ``CalcJob`` wrapping class for the code, filling the inputs necessary, and then running or submitting the constructed object.
 
+If the ``CalcJob`` has been registered in AiiDA (see section :ref:`on how to register plugins<how-to:plugins:entrypoints>`) you can use the ``CalculationFactory``:
+
+.. code-block:: python
+
+    from aiida.plugins import CalculationFactory
+    calculation_builder = CalculationFactory('arithmetic.add').get_builder()
+
+Otherwise, you can just load the class and get the builder directly:
+
+.. code-block:: python
+
+    from aiida.calculations.plugins.arithmetic.add import ArithmeticAddCalculation
+    calculation_builder = ArithmeticAddCalculation.get_builder()
```

Comment:
This will very often lead to problems when they start submitting to the daemon and I would not mention it here as it is an unnecessary alternative.